engine/access/rpc/backend/backend.go

```diff
@@ -76,6 +76,9 @@ type Backend struct {
 	collections       storage.Collections
 	executionReceipts storage.ExecutionReceipts
 	connFactory       connection.ConnectionFactory
+
+	sporkRootBlockHeight uint64
```

Comment:
yea, we definitely could. I cached it on the backend since they are small `uint64`, and never change. So they are easy to cache and save on the db lookups

Reply:
updated in [24898eb](https://github.com/onflow/flow-go/pull/4690/commits/24898eb809fa2b5b669b716a784d3bf6aba348d5)